EDI/Operation/05_ManageDataSources.md

```diff
@@ -0,0 +1,156 @@
+    
+    The *Attribute set* drop-down list is automatically filled with the data source defined in the *Entity type* field.
+    
+5. Click the [CONTINUE] button in the bottom right corner.   
+    The *Map attributes* pop-up window is displayed. If you have not yet defined string attributes in your file settings, the *There are no mappable fields in the destination* text is displayed. You can ignore this step for now.
+
+    ![Map attributes](../../Assets/Screenshots/EDI/Operation/DataSourcesMapAttributes.png "[Map attributes]")
+
+6. Click the [CONTINUE] button in the bottom right corner.   
+    The *Optional: Filter items* wizard window is displayed. You can ignore it. It is only relevant for mass transports.
+
+     ![Optional: Filter items](../../Assets/Screenshots/EDI/Operation/DataSourcesOptionalFilterItems.png "[Optional:Filter items]")
+
+7. Click the [CONTINUE] button in the bottom right corner.   
+    The *Select additional data* wizard window is displayed. 
+
+    ![Select additional data](../../Assets/Screenshots/EDI/Operation/DataSourcesSelectAdditionalData.png "[Select additional data]")
+
+8. If desired, specify a limit. The limit defines the maximum number of line items which are to be added to a business document such as an invoice. <!---Julian, ist das richtig?-->
+
+9. Click the [FINALIZE] button in the bottom right corner to save the data source.   
+   The *Exporter definition sources* view is displayed.
+
+10. Click the [SAVE] button in the top right corner.    
+   The *Submitting data* screen is displayed. After that, the *Exporter definitions* view is displayed again. 
+
+
+
+## Map data source
+
+Map the data sources by specifying the attributes that are to be mapped on EDIFACT message fields.
+
+#### Prerequisites
+
+- At least one data source has been assigned, see [Assign data source](#assign-data-source).
+- In the file settings, you have added at least one string attribute. See an example under [Specify document number &frasl; string attribute](./02_ManageHeaderFileSett.md#specify-document-number-⁄-string-attribute).
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Export definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Export definitions]")
+
+1. Click the exporter definition for which you want to define the data source.   
+    The *Edit definition "Definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+2. Click the *Data sources* tab.   
+    The *Data sources* tab is displayed.  
+
+    ![Data sources tab](../../Assets/Screenshots/EDI/Operation/DataSourcesListWithEntries.png "[Data sources tab]")
+
+3. Click the data source for which you want to map the attributes.   
+    The *Map attributes* pop-up window is displayed. In the *Destination attribute* column, a list of all EDI segment rows is displayed for which you have defined a string attribute.
```

Comment:
Vorschlag: a list of all EDI segment rows for which you have defined a string attribute is displayed.

Reply:
ne, Verb steht zu weit vom Subjekt